web_src/js/webcomponents/absolute-date.ts

```diff
@@ -1,5 +1,12 @@
-export function toAbsoluteLocaleDate(date: string, lang: string, opts: Intl.DateTimeFormatOptions) {
-  return new Date(date).toLocaleString(lang || [], opts);
+export function toAbsoluteLocaleDate(date: string, lang?: string, opts?: Intl.DateTimeFormatOptions) {
```

Comment:
The removals of the `?` on second and third argument is incorrect. I will fix it in https://github.com/go-gitea/gitea/pull/32308.

Reply:
Edit: nevermind, I see the PR corrected it actually.